README.md

```diff
@@ -47,6 +47,13 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
 <table>
     <tbody>
         <tr>
+        <td align="center">
+            <a href="https://github.com/pmutyalu">
+                <img src="https://avatars.githubusercontent.com/u/130680906?v=4" width="100px;" alt="P.Mutyalu"/>
+                <br />
+                <sub><b>P.Mutyalu</b></a>
+            </a>
+        </td>
```

Comment:
Please indent correctly (add four spaces to the beginning of every single line).

Reply:
![image](https://github.com/user-attachments/assets/05e3f7f6-97e7-4206-ba58-e86dbf472cce)
The text alignment in my IDE is ok, but idk why the text is coming into below row.